gazelle/kotlin/tests/test_targets/src/test/xyz/foo/BUILD.out

```diff
@@ -0,0 +1,9 @@
+kt_jvm_test(
+    name = "calculatortest_test",
+    srcs = ["CalculatorTest.kt"],
+    test_class = "xyz.foo.CalculatorTest",
+    deps = [
+        "@example//goog_truth",
```

Comment:
Needs an additional dep for `gazelle/kotlin/tests/test_targets/src/main/xyz/foo/Calculator.kt`.